numpy_tasks.ipynb

```diff
@@ -9,6 +9,25 @@
     "https://docs.google.com/spreadsheets/d/1v1_X6zRlwRVjULZSrRvBYAG_raUPjTI0Ceruwh6vNXw/edit#gid=2019320840\n"
```

Comment:
<p>Why do you use indexing for the first axis, but slicing for the second one?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bestetc/May/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
<p>There are no matter using the indexing on the first axis in this symmetric case.</p><p>Important uses the indexing and slicing both.</p>

Reply:
<p>Can be performed only using slices, fix please</p>